static/js/containers/LearnerSearchPage.js

```diff
@@ -76,15 +80,15 @@ class LearnerSearchPage extends React.Component {
   };
 
   closeEmailComposerAndSend = (): void => {
-    const { dispatch, searchResultEmail: { inputs, params } } = this.props;
+    const { dispatch, searchResultEmail: { inputs } } = this.props;
     let errors = emailValidation(inputs);
     dispatch(updateEmailValidation({type: SEARCH_EMAIL_TYPE, errors: errors}));
     if (R.isEmpty(errors)) {
       dispatch(
         sendSearchResultMail(
           inputs.subject || '',
           inputs.body || '',
-          params.searchkit.buildQuery().query
+          this.searchkit.buildQuery().query
```

Comment:
These changes are essential in getting the tests to pass, though. But I'm happy to wait until #2555 is merged then rebase on that